deepsearch/documents/core/utils.py

```diff
@@ -144,11 +217,21 @@ def download_url(url: str, save_path: Path, chunk_size=128):
     return
 
 
-def get_urls(url_path: Path) -> List[str]:
+def read_lines(file_path: Path) -> List[str]:
     """
-    Returns list of url from input file.
+    Returns list of lines from input file.
     """
 
-    lines = url_path.read_text()
-    urls = [line.strip() for line in lines.split("\n") if line.strip() != ""]
-    return urls
+    lines = file_path.read_text()
+    line = [line.strip() for line in lines.split("\n") if line.strip() != ""]
+    return line
+
+
+def write_taskids(result_dir: Path, list_to_write: List[str]) -> None:
```

Comment:
This method has only _one_ caller. I can't see why the code would deserve an own method, since it does nothing except opening a file and writing a list.